contracts/ERC721A.sol

```diff
@@ -395,29 +395,6 @@ contract ERC721A is Context, ERC165, IERC721, IERC721Metadata, IERC721Enumerable
         emit Approval(owner, to, tokenId);
     }
 
-    uint256 public nextOwnerToExplicitlySet = 0;
-
-    /**
-     * @dev Explicitly set `owners` to eliminate loops in future calls of ownerOf().
-     */
-    function _setOwnersExplicit(uint256 quantity) internal {
-        uint256 oldNextOwnerToSet = nextOwnerToExplicitlySet;
-        require(quantity > 0, "quantity must be nonzero");
-        uint256 endIndex = oldNextOwnerToSet + quantity - 1;
-        if (endIndex > currentIndex - 1) {
-            endIndex = currentIndex - 1;
-        }
-        // We know if the last one in the group exists, all in the group exist, due to serial ordering.
-        require(_exists(endIndex), "not enough minted yet for this cleanup");
```

Comment:
This line will never evaluate to true due to line 408 so I removed this check in the refactored function